sigs.yaml

```diff
@@ -0,0 +1,25 @@
+sigs:
+- dir: sig-operations
+  name: Operations
+  mission_statement: >
+    The goal of this SIG is to enable management and operations for hardware and platform tooling within the Operate
+    First Community Cloud (op1st)
+
+  charter_link: charter.md
+  label: operations
+  leadership:
+    chairs:
+    - github: tbd  # TODO: fill in
+      name: tbd  # TODO: fill in
+      company: Red Hat
```

Comment:
I nominate @HumairAK and @larsks (since there should be 2 chairs). Would you guys accept this proposal?

Reply:
Sure :)

Reply:
I added these as placeholers for now to allow for generator to work w/o errors. They are not necessarily final, we can discuss further here or in next meeting.